python/py-sentencepiece/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        google sentencepiece 0.1.84 v
+revision            0
+name                py-${github.project}
+categories          python textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python wrapper for ${github.project}
+
+long_description    SentencePiece is an unsupervised text tokenizer\
+                    and detokenizer mainly for Neural Network-based\
+                    text generation systems where the vocabulary size\
+                    is predetermined prior to the neural model\
+                    training. SentencePiece implements subword units\
+                    (e.g., byte-pair-encoding (BPE) \[Sennrich et al.\])\
+                    and unigram language model \[Kudo.\]) with the\
+                    extension of direct training from raw\
+                    sentences. SentencePiece allows us to make a\
+                    purely end-to-end system that does not depend on\
+                    language-specific pre/postprocessing.
+
+homepage            https://github.com/google/sentencepiece
+
+checksums           rmd160  91bc4a0a23d9a4daf98abbb9ec83aed2241fdc42 \
+                    sha256  dacab5fe63875b94538e06f4995ffb8cf35400708f3c333263f7b72e2e5b43de \
+                    size    11828855
+
+python.versions     37 38
+
```

Comment:
when compiling the extension it adds ```-stdc=c++11```, so you likely want to add ```compiler.cxx_standard 2011``` here as well

Reply:
Thanks done.